File: csharp/dsf/dsf/src/core/TaskQueueOverflowException.cs
using System;

namespace com.baidu.dsf.core
{

    /// 
    /// <summary>
    /// @author arksea
    /// </summary>
    public class TaskQueueOverflowException : Exception
    {

        public TaskQueueOverflowException(string name) : base(name)
        {
        }

        public TaskQueueOverflowException(string name, Exception ex) : base(name, ex)
        {
        }
    }

}

File: java/dsframework/src/main/java/dsf/core/TaskQueueOverflowException.java
package dsf.core;

/**
 *
 * @author arksea
 */
public class TaskQueueOverflowException extends RuntimeException {

    public TaskQueueOverflowException(String name) {
        super(name);
    }

    public TaskQueueOverflowException(String name, Throwable ex) {
        super(name, ex);
    }
}
